terraform/scripts/manually-upload-certs.sh

```diff
@@ -2,8 +2,8 @@
 
 set -eu
 
-export PASSWORD_STORE_DIR=${CERT_PASSWORD_STORE_DIR}
-
+export PASSWORD_STORE_DIR=~/.paas-pass-high
```

Comment:
Actually that's rubbish because in my case `pass certs/` doesn't exist - it's at `pass paas/high/certs/`.

I don't see a good solution to this, so I'm happy to temporarily set `PASSWORD_STORE_DIR` to run this.